src/main/java/microsoft/exchange/webservices/data/core/EwsUtilities.java

```diff
@@ -600,11 +545,11 @@ public static String formatLogMessage(String entryKind, String logEntry)
     XMLOutputFactory factory = XMLOutputFactory.newInstance();
     XMLStreamWriter writer = factory.createXMLStreamWriter(outStream);
     EwsUtilities.writeTraceStartElement(writer, entryKind, false);
-    writer.writeCharacters(System.getProperty("line.separator"));
+    writer.writeCharacters(SystemUtils.LINE_SEPARATOR);
```

Comment:
> The line.separator System Property. Line separator ("\n" on UNIX).
Defaults to null if the runtime does not have security access to read this property or the property does not exist.
**This value is initialized when the class is loaded.** If System.setProperty(String,String) or System.setProperties(java.util.Properties) is called after this class is loaded, the value will be out of sync with that System property.

Maybe this can be a potential bug if the `LINE_SEPARATOR` is changed afterwards.